<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(128, 0, 2);
font: 75 20pt &quot;Times&quot;;
</string>
    </property>
    <property name="text">
     <string>Input directory path here:</string>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(102, 204, 255);
gridline-color: rgb(102, 204, 255);
color: rgb(253, 128, 8);</string>
    </property>
    <property name="text">
     <string>Start Judge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Times New Roman&quot;;
background-color: rgb(179, 179, 179);</string>
    </property>
    <property name="text">
     <string>Equal Result:</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>801</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 180);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Times New Roman&quot;;
background-color: rgb(179, 179, 179);</string>
    </property>
    <property name="text">
     <string>Unequal Result:</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>801</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255, 190);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(102, 204, 255);
gridline-color: rgb(102, 204, 255);
color: rgb(253, 128, 8);</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>Maual Judge</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionCheck_History"/>
   </widget>
   <widget class="QMenu" name="menuJudeg">
    <property name="title">
     <string>Judge</string>
    </property>
    <addaction name="actionAuto_judge"/>
    <addaction name="actionManual_judge"/>
   </widget>
   <widget class="QMenu" name="menuPersonal_Info">
    <property name="title">
     <string>Personal Info</string>
    </property>
    <addaction name="actionInformation"/>
    <addaction name="actionLogOut"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuJudeg"/>
   <addaction name="menuPersonal_Info"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionCheck_History">
   <property name="text">
    <string>Check History</string>
   </property>
  </action>
  <action name="actionAuto_judge">
   <property name="text">
    <string>Auto-judge</string>
   </property>
  </action>
  <action name="actionManual_judge">
   <property name="text">
    <string>Manual-judge</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionLogOut">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
